README.md

```diff
@@ -17,20 +17,21 @@ sudo skopeo copy containers-storage:localhost/thoth-python-36:latest docker://qu
+| thoth-pipeline-github  | current    |
+| thoth-precommit-py39   | current    | [![Docker Repository on Quay](https://quay.io/repository/thoth-station/thoth-precommit-py39/status "Docker Repository on Quay")](https://quay.io/repository/thoth-station/thoth-precommit-py39)     |
+| thoth-precommit-py38   | current    | [![Docker Repository on Quay](https://quay.io/repository/thoth-station/thoth-precommit-py38/status "Docker Repository on Quay")](https://quay.io/repository/thoth-station/thoth-precommit-py38)     |
+| thoth-precommit-py36   | current    | [![Docker Repository on Quay](https://quay.io/repository/thoth-station/thoth-precommit/status "Docker Repository on Quay")](https://quay.io/repository/thoth-station/thoth-precommit)               |
+| thoth-pytest-py38      | current    | [![Docker Repository on Quay](https://quay.io/repository/thoth-station/thoth-pytest-py38/status "Docker Repository on Quay")](https://quay.io/repository/thoth-station/thoth-pytest-py38)           |
+| thoth-pytest-py36      | current    | [![Docker Repository on Quay](https://quay.io/repository/thoth-station/thoth-pytest/status "Docker Repository on Quay")](https://quay.io/repository/thoth-station/thoth-pytest)                     |
+| thoth-pylint-ubi8      | current    | [![Docker Repository on Quay](https://quay.io/repository/thoth-station/thoth-pylint/status "Docker Repository on Quay")](https://quay.io/repository/thoth-station/thoth-pylint)                     |
+| thoth-pytest-ubi8-py38 | deprecated | [![Docker Repository on Quay](https://quay.io/repository/thoth-station/thoth-pytest-ubi8-py38/status "Docker Repository on Quay")](https://quay.io/repository/thoth-station/thoth-pytest-ubi8-py38) |
+| thoth-mypy-ubi8        | deprecated | [![Docker Repository on Quay](https://quay.io/repository/thoth-station/thoth-mypi/status "Docker Repository on Quay")](https://quay.io/repository/thoth-station/thoth-mypi)                         |
+| thoth-pytest-f31-py37  | deprecated | [![Docker Repository on Quay](https://quay.io/repository/thoth-station/thoth-pytest-f31-py37/status "Docker Repository on Quay")](https://quay.io/repository/thoth-station/thoth-pytest-f31-py37)   |
+| thoth-black-ubi8       | deprecated | [![Docker Repository on Quay](https://quay.io/repository/thoth-station/thoth-black/status "Docker Repository on Quay")](https://quay.io/repository/thoth-station/thoth-black)                       |
+| thoth-coala-ubi8       | deprecated | [![Docker Repository on Quay](https://quay.io/repository/thoth-station/thoth-coala/status "Docker Repository on Quay")](https://quay.io/repository/thoth-station/thoth-coala)                       |
+| thoth-src              | deprecated | [![Docker Repository on Quay](https://quay.io/repository/thoth-station/thoth-srcops/status "Docker Repository on Quay")](https://quay.io/repository/thoth-station/thoth-srcops)                     |
```

Comment:
That's for readme rendering with a table right ? Does the current works fine ?

Reply:
ja.